CONTRIBUTING.rst

```diff
@@ -21,7 +21,7 @@ In order to add a feature to ``google-cloud-python``:
   documentation (in ``docs/``).
 
 - The feature must work fully on the following CPython versions:  2.7,
-  3.4, and 3.5 on both UNIX and Windows.
+  3.4, 3.5, and 3.6 on both UNIX and Windows.
```

Comment:
FWIW, The 3.4 release line is in ["security fixes only" mode](http://permalink.gmane.org/gmane.comp.python.committers/4809), and not sunsetted (as 3.3 is about to be).